scripts/ci/reject_master_prs.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -ev
+
+! [[ $TRAVIS_PULL_REQUEST != 'false' && $TRAVIS_BRANCH == 'master' ]]
```

Comment:
If this fails, we should display a useful error in the logs. Otherwise people will just reach out to us with questions. We can even include the instructions to retarget the PR to dev--the goal being to avoid receiving direct pings via Teams or whatever.

Reply:
Fixed in 6277132